CONTRIBUTING.md

```diff
@@ -86,7 +86,7 @@ To contribute a patch, the workflow is as follows:
   1. Commit patches
 
 For GUI-related issues or pull requests, the https://github.com/bitcoin-core/gui repository should be used.
-For all other issues and pull requests, the https://github.com/bitcoin/bitcoin node repository should be used.
```

Comment:
Currently, it's recommended to open PRs to bitcoin/bitcoin when possible, and only on bitcoinknots/bitcoin when the PR is inherently exclusive to Knots.